package-lock.json

```diff
@@ -0,0 +1,5775 @@
+{
```

Comment:
Why has a package-lock been introduced?. We use yarn?

Reply:
Looks like I introduced it in the first commit accidentally.